covid_moonshot/analysis/fah_plotting.py

```diff
@@ -76,49 +60,73 @@ def plot_relative_distribution(relative_fes, bins=100, title='Relative affinity'
 
     """
     # first convertt kT to kcal/mol
-    
-    relative_fes = [(x*kT).value_in_unit(unit.kilocalories_per_mole) for x in relative_fes]
-    sns.kdeplot(
-        relative_fes, shade=True, color="hotpink")
+
+    relative_fes = [
+        (x * kT).value_in_unit(unit.kilocalories_per_mole) for x in relative_fes
+    ]
+    sns.kdeplot(relative_fes, shade=True, color="hotpink")
     sns.rugplot(
-        relative_fes,
-        color="hotpink",
-        alpha=0.5,
-        label=f"N={len(relative_fes)}",
+        relative_fes, color="hotpink", alpha=0.5, label=f"N={len(relative_fes)}",
     )
     plt.xlabel("Relative free energy to ligand 0 / kcal/mol")
     plt.title(title)
-    _produce_plot(filename)
 
 
 def plot_convergence(results, n_gens=3, title=None):
```

Comment:
@hannahbrucemacdonald @jchodera it looks like this plot relies on results at the GEN level, which we are not yet computing in the analysis pipeline. Is it worth adding this analysis, or better to skip this plot for now?

Reply:
Let's break out the GEN-level info into a separate issue to first discuss how to represent work values can be organized by GEN. That's something that will be important as a sanity check.